api/clients/accountant_test.go

```diff
@@ -311,33 +312,33 @@ func TestAccountBlob_ReservationWithOneOverflow(t *testing.T) {
 	now := time.Now().Unix()
 
 	// Okay reservation
```

Comment:
do we need equality assertions on salt values?

Reply:
I added some assertions for coverage but didn't add it throughout, lmk if you think that will be useful